### PR TITLE
[examples] Fix hackernews service

### PR DESCRIPTION
**Pull request by skiplabsdaniel**

### CODE REVIEW

File: examples/hackernews/back-end/reactive_service/src/hackernews.service.ts [hunk 1]
```diff
@@ -85,25 +91,22 @@ class PostsCleanupKeyMapper {
     it: NonEmptyIterator<TJSON>,
   ): Iterable<[string, TJSON]> {
     const post = it.first();
-    return [[key[1].toString(), post]];
+    return [[key[1].toString(), post] as [string, TJSON]];
```
Comment (beauby) [inline examples/hackernews/back-end/reactive_service/src/hackernews.service.ts]: Why is this needed?

Comment (skiplabsdaniel) [reply examples/hackernews/back-end/reactive_service/src/hackernews.service.ts]: No, just due to intermediate error checking, I remove it.

File: examples/hackernews/back-end/reactive_service/src/hackernews.service.ts [hunk 2]
```diff
@@ -71,11 +72,16 @@ class PostsMapper {
...
     it: NonEmptyIterator<Post>,
-  ): Iterable<[[number, number], Post & { upvotes: number; author: User }]> {
+  ): Iterable<[[number, number], Upvoted]> {
     const post = it.first();
     const upvotes = this.upvotes.getArray(key).length;
-    const author = this.users.getOne(post.author_id);
-    return [[[-upvotes, key], { ...post, upvotes, author }]];
+    const author = this.users.maybeGetOne(post.author_id);
+    return [
+      [[-upvotes, key], { ...post, upvotes, author }] as [
```
Comment (beauby) [inline examples/hackernews/back-end/reactive_service/src/hackernews.service.ts]: The `as` shouldn't be needed here either.

Comment (skiplabsdaniel) [reply examples/hackernews/back-end/reactive_service/src/hackernews.service.ts]: Yes
